.github/workflows/build-m1-wheel.yml

```
@@ -29,29 +29,44 @@ jobs:
       with:
         fetch-depth: 0
 
-    - name: Lint source with flake8
+    - name: Brew update Python
       run: |
-        arch -arm64 python -m pip install --upgrade pip
-        arch -arm64 pip install flake8
-        arch -arm64 flake8 src setup.py python-bindings python-impl
+        arch -arm64 brew install python@3.9
+        arch -arm64 brew install python@3.10
 
-    - name: Lint source with mypy
+    - name: Build m1 wheels for 3.9 and 3.10
       run: |
-        arch -arm64 pip install mypy
-        arch -arm64 mypy --config-file mypi.ini python-bindings python-impl
-
-    - name: Build m1 wheels
-      run: |
-        arch -arm64 pip wheel -w dist --no-deps .
-        arch -arm64 delocate-wheel -v dist/*.whl
+        rm -rf venv
+        arch -arm64 /opt/homebrew/opt/python@3.9/libexec/bin/python -m venv venv
```

```suggestion
        arch -arm64 $(brew --prefix python@3.9)/libexec/bin/python -m venv venv
```

I think we've been trying to get the prefix dynamically since it changes to differently locations on Intel vs M1 macs - its easier to port something to work on any of them without necessarily knowing the details of where homebrew puts these things

---

Smart. Adding it globally.